src/ClientData/ProcessData.cpp

```diff
@@ -135,17 +135,21 @@ ErrorMessageOr<ModuleInMemory> ProcessData::FindModuleByAddress(uint64_t absolut
                         absolute_address, process_info_.name()));
   }
 
-  ErrorMessage not_found_error = ErrorMessage(absl::StrFormat(
-      "Unable to find module for address %016x: No module loaded at this address by process %s",
-      absolute_address, process_info_.name()));
-
   auto it = start_address_to_module_in_memory_.upper_bound(absolute_address);
-  if (it == start_address_to_module_in_memory_.begin()) return not_found_error;
+  if (it == start_address_to_module_in_memory_.begin()) {
+    return ErrorMessage{absl::StrFormat(
+        "Unable to find module for address %016x: No module loaded at this address by process %s",
```

Comment:
Can we still extract that template string as a constant?

Reply:
I had thought about it but I was undecided. Good to have an opinion. Done.